exercises/reverse-string/example.jl

```diff
@@ -0,0 +1,12 @@
+function myreverse(phrase::String)
+    chars = []
+    for i in phrase
+        push!(chars, i)
+    end
+    newchars=chars[end:-1:1]
+    newphrase= ""
+    for j in newchars
+        newphrase = "$newphrase$j"
+    end
```

Comment:
This is really slow because you create new strings over and over again. You could use `join(newchars)` to create a string from an array of chars.